crates/wasmer-pack/src/pirita.rs

```diff
@@ -39,22 +41,44 @@ fn libraries(webc: &WebC<'_>, fully_qualified_package_name: &str) -> Result<Vec<
     let Manifest { bindings, .. } = webc.get_metadata();
     let libraries = bindings
         .iter()
-        .map(|b| load_library(b, webc, fully_qualified_package_name))
+        .map(|b| load_library(b.clone(), webc, fully_qualified_package_name))
         .collect::<Result<Vec<_>, _>>()?;
 
     Ok(libraries)
 }
 
-fn metadata(fully_qualified_package_name: &str) -> Result<Metadata, Error> {
+#[derive(Debug, Serialize, Deserialize)]
+struct InternalPackageMeta {
+    description: String,
+}
+
+/// Returns the wapm package description.
+///
+/// Similar to `WebC::get_package_name_from_manifest` in the webc crate.
+/// This should probably be in the webc crate as well,
+/// but I don't want to mess with dependencies.
+///
+/// TODO: This function alone adds a new dependency to the project: `serde_cbor`.
+///       Move this function to a better place, most likely the webc crate
+fn get_description_from_webc_manifest(m: &Manifest) -> String {
+    let wapm: Option<webc::metadata::annotations::Wapm> = m.package_annotation("wapm").unwrap();
+    match wapm {
+        None => "".to_string(),
+        Some(wapm) => wapm.description,
+    }
+}
```

Comment:
I'd prefer not to get the description this way because it means 1) we'll crash when no `"wapm"` annotation is present (the `.unwrap()`), and 2) we're using the empty string to represent the absence of a description, when Rust traditionally prefers `None`.

We should be able to solve both problems in one line using pattern matching and `map()`.

```suggestion
/// Returns the wapm package description.
///
/// Similar to `WebC::get_package_name_from_manifest` in the webc crate.
/// This should probably be in the webc crate as well,
/// but I don't want to mess with dependencies.
fn get_description_from_webc_manifest(m: &Manifest) -> Option<String> {
    m.package_annotation("wapm")
        .map(|Wapm { description, .. }| description)
}
```

That does mean you'll need to do something like this when updating the metadata:

```
let mut metadata = Metadata::new(package_name, version);

if let Some(description) = get_description_from_webc_manifest(m) {
  metadata = metadata.with_description(description);
}

Ok(metadata)
```

That means the description is still not guaranteed to be set, though. If PyPI requires the `description` to be set then we should probably error out (or maybe emit a warning?) at the top of `wasmer_pack::py::generate_python()`, because that's a constraint specific to the Python generator.